apertium-dan.dan.dix

```diff
@@ -60688,6 +60688,961 @@ TODO:
+<e lm="irreligiøs">                    <i>irreligiøs</i><par n="klar__adj"/></e>
+<e lm="islagt">                        <i>islagt</i><par n="stolt__adj"/></e>
+<e lm="janfri">                        <i>janfri</i><par n="klar__adj"/></e>
+<e lm="jasket">                        <i>jaske</i><par n="øve/t__adj"/></e>
+<e lm="jordbunden">                    <i>jordbund</i><par n="besked/en__adj"/></e>
+<e lm="jordet">                        <i>jorde</i><par n="øve/t__adj"/></e>
+<e lm="jovial">                        <i>jovial</i><par n="klar__adj"/></e>
+<e lm="judiciel">                      <i>judiciel</i><par n="finansiel__adj"/></e>
+<e lm="jævnbred">                      <i>jævnbred</i><par n="klar__adj"/></e>
+<e lm="kadetstribet">                  <i>kadetstribe</i><par n="øve/t__adj"/></e>
+<e lm="kalkfri">                       <i>kalkfri</i><par n="klar__adj"/></e>
+<e lm="kamtakket">                     <i>kamtakke</i><par n="øve/t__adj"/></e>
+<e lm="kanariegul">                    <i>kanariegul</i><par n="klar__adj"/></e>
+<e lm="kanonfuld">                     <i>kanonfuld</i><par n="klar__adj"/></e>
+<e lm="kapriciøs">                     <i>kapriciøs</i><par n="klar__adj"/></e>
+<e lm="kardial">                       <i>kardial</i><par n="klar__adj"/></e>
+<e lm="kariøs">                        <i>kariøs</i><par n="klar__adj"/></e>
+<e lm="kategorial">                    <i>kategorial</i><par n="klar__adj"/></e>
+<e lm="kejthåndet">                    <i>kejthånde</i><par n="øve/t__adj"/></e>
+<e lm="kemikaliefri">                  <i>kemikaliefri</i><par n="klar__adj"/></e>
+<e lm="kernefuld">                     <i>kernefuld</i><par n="klar__adj"/></e>
+<e lm="kilden">                        <i>kild</i><par n="besked/en__adj"/></e>
+<e lm="kimfri">                        <i>kimfri</i><par n="klar__adj"/></e>
+<e lm="kisteglad">                     <i>kisteglad</i><par n="stolt__adj"/></e>
+<e lm="klangfuld">                     <i>klangfuld</i><par n="klar__adj"/></e>
+<e lm="klarhovedet">                   <i>klarhovede</i><par n="øve/t__adj"/></e>
+<e lm="klassebevidst">                 <i>klassebevidst</i><par n="stolt__adj"/></e>
+<e lm="kliddet">                       <i>klidde</i><par n="øve/t__adj"/></e>
+<e lm="klimakteriel">                  <i>klimakteriel</i><par n="finansiel__adj"/></e>
+<e lm="klippefuld">                    <i>klippefuld</i><par n="klar__adj"/></e>
+<e lm="klokkeklar">                    <i>klokkeklar</i><par n="klar__adj"/></e>
+<e lm="klumpfodet">                    <i>klumpfode</i><par n="øve/t__adj"/></e>
+<e lm="knastfri">                      <i>knastfri</i><par n="klar__adj"/></e>
+<e lm="kneben">                        <i>kneb</i><par n="besked/en__adj"/></e>
+<e lm="knebet">                        <i>kneb</i><par n="besked/en__adj"/></e>
+<e lm="knoldet">                       <i>knolde</i><par n="øve/t__adj"/></e>
```

Comment:
wrong pardef? lemma is -en, pardef is -en

Reply:
"knebet" skulle ikke have været med